### PR TITLE
remove AWS/Azure/GCP BSL/VSL config tables from docs

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <krisss@vmware.com>

PR 1 of 4 for #2003, will also submit a PR per plugin repo to remove common docs sections

cc @carlisia @nrb @prydonius

### CODE REVIEW

File: site/docs/master/supported-providers.md [hunk 1]
```diff
@@ -4,20 +4,21 @@ Velero supports a variety of storage providers for different backup and snapshot
...
 ## Velero supported providers
 
-| Provider                   | Object Store        | Volume Snapshotter           | Plugin                    |
-|----------------------------|---------------------|------------------------------|---------------------------|
-| [AWS S3][7]                | AWS S3              | AWS EBS                      | [Velero plugin AWS][8]    |
-| [Azure Blob Storage][9]    | Azure Blob Storage  | Azure Managed Disks          | [Velero plugin Azure][10] |
-| [Google Cloud Storage][11] | Google Cloud Storage| Google Compute Engine Disks  | [Velero plugin GCP][12]   |
+| Provider                          | Object Store        | Volume Snapshotter           | Plugin Documentation      |
+|-----------------------------------|---------------------|------------------------------|---------------------------|
+| [Amazon Web Services (AWS)][7]    | AWS S3              | AWS EBS                      | [Velero plugin AWS][8]    |
```
Comment (carlisia) [inline site/docs/master/supported-providers.md]: Hm. I didn't originally add the `for` but looking at it now I think we should. Ex; `Velero plugin for AWS`.

Comment (skriss) [reply site/docs/master/supported-providers.md]: sure, I can add that

**Comment by prydonius** [comment]
Did we want to retroactively apply this for the 1.2.0-beta.1 docs?

**Comment by skriss** [comment]
@prydonius we don't need to - I'll regenerate the docs for v1.2 to include this, and the beta docs will be removed.

**Pull request merged**